src/System.Management.Automation/engine/Modules/GetModuleCommand.cs

```diff
@@ -375,10 +375,9 @@ protected override void ProcessRecord()
             var moduleSpecTable = new Dictionary<string, ModuleSpecification>(StringComparer.OrdinalIgnoreCase);
             if (FullyQualifiedName != null)
             {
-                // TODO:
-                // FullyQualifiedName.Name could be a path, in which case it will not match module.Name.
-                // This is potentially a bug (since version checks are ignored).
-                // We should normalize FullyQualifiedName.Name here with ModuleIntrinsics.NormalizeModuleName().
+                FullyQualifiedName = FullyQualifiedName.Select(
```

Comment:
It looks like we already have some LINQ usage here, but if we can avoid it I'd like to. As nice as it is, the GC pressure it creates is a performance drain.